.env.example

```diff
@@ -24,6 +24,7 @@ PROXY_ACCOUNT_CLASS_HASH=0xba8f3f34eb92f56498fdf14ecac1f19d507dcc6859fa6d85eb854
 
 ## configurations for testing
 COMPILED_KAKAROT_PATH=lib/kakarot/build
+FOUNDRY_OUTPUT_PATH=./lib/kakarot/tests/integration/solidity_contracts/build
```

Comment:
do we need `./` or not? see it's not put for `COMPILED_KAKAROT_PATH` ; when adding similar context into a file, make sure that you stay consistent